src/pipelines/mod.rs

```diff
@@ -109,6 +129,7 @@ impl TrunkLink {
 pub enum TrunkLinkPipelineOutput {
```

Comment:
```suggestion
pub enum TrunkAssetPipelineOutput {
```

I'm thinking that we should go ahead and rename this to match the other items you've renamed as well. I think the rename is quite tasteful. Semantically it is much more meaningful, I would say.